### PR TITLE
Dockerfile vs java_image example

### PR DESCRIPTION
**Pull request by alex1545**

### CODE REVIEW

File: testing/examples/java_app/image_data/Greeting.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// package com.example;
```
Comment (nlopezgi) [inline testing/examples/java_app/image_data/Greeting.java]: add  license header

File: testing/examples/java_app/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FROM openjdk
```
Comment (nlopezgi) [inline testing/examples/java_app/Dockerfile]: add brief doc

Comment (alex1545) [reply testing/examples/java_app/Dockerfile]: Done

File: testing/examples/java_app/image_data/Greeting.java (hunk 1 above)
Comment (alex1545) [reply testing/examples/java_app/image_data/Greeting.java]: Done

**Pull request merged**